catroid/src/main/res/values/strings.xml

```diff
@@ -1049,6 +1049,8 @@
     <string name="brick_speak">Speak</string>
     <string name="brick_speak_default_value">Hello!</string>
     <string name="brick_speak_and_wait">and wait</string>
+    <string name="brick_start_listening_to_voice">Start listening to voice and store spoken words in</string>
+    <string name="no_speech_recognition_available">Speech Recognition isn\'t available</string>
```

Comment:
```suggestion
    <string name="no_speech_recognition_available">The project uses speech recognition, but no speech recognition for the selected language is available. Please add a speech recognizer via your system settings.</string>
```

Reply:
I guess you misunderstood this error msg, this tells the user that his/her smartphone doesn't have a speech recognition engine, furthermore, it can't be downloaded.(has nothing to do with the chosen language)